js_modules/dagit/packages/ui/src/components/SplitPanelContainer.tsx

```diff
@@ -15,6 +15,7 @@ interface SplitPanelContainerProps {
   firstInitialPercent: number;
   firstMinSize?: number;
   second: React.ReactNode | null; // Note: pass null to hide / animate away the second panel
+  height?: string;
```

Comment:
See comment on the `<SplitPanelContainer>` callsite, I'm not sure you need this.